align/align.go

```diff
@@ -201,11 +200,12 @@ func SmithWaterman(stringA string, stringB string, scoring Scoring) (int, string
 }
 
 func score(a, b byte, scoring Scoring) int {
-	if a == b {
-		return scoring.Match
-	} else {
-		return scoring.Mismatch
+	currentScore, err := scoring.SubstitutionMatrix.Score(string(a), string(b))
+	if err != nil {
+		panic(err)
```

Comment:
worried that panic will kill large alignment runs. Is there a better way to handle errors here @ragnorc?